PageEdit/PageEdit.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Edit info</title>
+  <link rel="stylesheet" type="text/css" href="PageEdit.css" />
+</head>
+
+<body>
```

Comment:
**Read about semantic html and refactor your markup (html) such that it reflects that semantic elements are used in the markup with proper hierarchical breakdown.**